term/sixel.go

```diff
@@ -40,6 +39,13 @@ func checkSixel() bool {
 	if err != nil {
 		return false
 	}
+	// Check tmux (sixel enabled)
+	// See: CSI Ps c, Ps = 4 (Sixel graphics)
+	// https://invisible-island.net/xterm/ctlseqs/ctlseqs.html
+	// NOTE: tmux does not return VT name.
+	if bytes.HasPrefix(b[:n], []byte("\x1b[?1;2;4c")) {
```

Comment:
why not adding this prefix to the `supportedTerminals` list?

Reply:
`1;2;` is not terminal name like `62` (vt200) or `63` (vt300).